assets/js/googlesitekit/widgets/datastore/areas.js

```diff
@@ -118,53 +119,46 @@ export const actions = {
 
 export const controls = {};
 
-export const reducer = ( state, { type, payload } ) => {
+export const reducer = createReducer( ( draft, { type, payload } ) => {
```

Comment:
I was contemplating this too. There's an advantage to calling it `draft` though, as we have a mixture of Immer and non-Immer reducers as it helps to distinguish which type of reducer you're looking at.

At the same time it's a bit of a generic term and doesn't describe what it is, i.e. "state".

A third option could be `draftState` - clearly states what it is and also distinguishes from the vanilla reducers. Looks like there are plenty of people using the name `draftState` on GitHub, FWIW :0) - https://github.com/search?q=draftstate&type=code

Reply:
Personally I'd prefer `state` over `draftState` because we'll have mixed reducers for awhile and it makes it easier to scan code for state. The enclosing `createReducer` code signifies it's safe, and I'd rather us write an ESLint rule to warn against modifying a variable named `state` when _not_ inside a `createReducer` function over naming everything `draftState` only to later—presumably—rename everything.

Given you're open to `state` already—and it's the most accurate term for what it is—let's go with that for now 😏

Reply:
`state` is fine by me :+1:

Reply:
Thanks, @tofumatt & @techanvil for this reasonable suggestion. I change to 'state' now 😼